Makefile

```diff
@@ -24,20 +25,17 @@ release:
 release-local:
 	goreleaser release --rm-dist --snapshot --skip-publish -f ./build/ci/goreleaser.yml
 
-TEMPLATES_API_BUILD_TARGET_PATH ?= "./target/templates/api"
 build-templates-api:
-	CGO_ENABLED=0 go build -v -mod mod -ldflags "-s -w" -o $(TEMPLATES_API_BUILD_TARGET_PATH) ./cmd/templates-api
+	CGO_ENABLED=0 go build -v -mod mod -ldflags "-s -w" -o "$(or $(BUILD_TARGET_PATH), ./target/templates/api)" ./cmd/templates-api
```

Comment:
Makefile nepodporuje args pre commandy, vsetko sa riesi cez ENVs.

Zjednotil som nazvy roznych ENVs, ktore definovali, kam sa ma ulozit skompilovana binarka.
Teraz je to uz iba jedna ENV `BUILD_TARGET_PATH`.